modules/common/src/errors.ts

```diff
@@ -153,3 +153,24 @@ export class DataDecodingError extends Error {
     super("Cannot decode data: " + message);
   }
 }
+
+export class EnsUnsuportedByNetworkError extends Error {
```

Comment:
Aren't e providing our own? When is this used?

Reply:
yes this is an edge case, where for some reason the ensAddress is empty in the provider and we dont have one to fill it with, we can delete the check if it does not make sense to you

Reply:
Not sure I follow. If we provide our own ENS always, would such an error ever be used?

Reply:
I just used it in one check, changed the error to 'UnsupportedNetworkError'